src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -3652,18 +3652,17 @@ static LogicalResult verify(ONNXDepthToSpaceOp op) {
     return op.emitError("Input should have a rank of four");
 
   // Check blocksize.
-  APInt blocksize = operandAdaptor.blocksize().getValue();
-  if (blocksize.isNegative())
+  int64_t blocksize = operandAdaptor.blocksize();
```

Comment:
So they changed the convention for the named parameters, they return now the int? Good to know

Reply:
yes, one of the several API changes that went into MLIR.